diagnostic.md

```diff
@@ -6,21 +6,25 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    The ember application router tells ember how to connect urls with specific view-states -
+    the main tasks you perform there are setting up those connections and specifying which
+    urls should link to which view-states. The main tasks you perform inside an ember Route
+    are specific to the individual model being dealt with at the time, primarily setting up a
+    model hook and determining how to deal with ids passed in through urls.
```

Comment:
Yes, the Ember Application Router defines the route, and the Ember Route specifies how to load model data using the model hook.